web/js/components/location-search/coordinates-dialog.js

```diff
@@ -14,14 +14,23 @@ class CoordinatesDialog extends Component {
       showTooltips: true,
       isCopyToClipboardTooltipVisible: false,
     };
+    this.copyToClipboard = this.copyToClipboard.bind(this);
   }
 
-  // set copy tooltip time
-  onCopyToClipboard = () => {
-    this.setState({
-      tooltipToggleTime: Date.now(),
-      isCopyToClipboardTooltipVisible: true,
-    });
+  copyToClipboard(coords) {
+    const options = window.clipboardData ? {} : { format: 'text/plain' };
+    options.onCopy = () => {
+      this.setState({
+        tooltipToggleTime: Date.now(),
+        isCopyToClipboardTooltipVisible: true,
+      });
+
+      // Prevnet keyboard overlay in iOS
+      setTimeout(() => {
+        document.getElementById('geosearch-autocomplete').blur();
```

Comment:
```suggestion
        document.getElementById('location-search-autocomplete').blur();
```
Here is the updated id name

Reply:
Ah, good catch.  I guess I wasn't up to date with develop